values.yaml

```diff
@@ -104,3 +104,52 @@ oda:
   use_hostpath: true
   ddaQueue: queue-osa11
   ddcache: "/net/cdcicn02/scratch/shared/savchenk/data/reduced/ddcache"
+
+plugins:
+  osa:
+    ebabled: true
+    instruments:
+      isgri:
+        dispatcher_mnt_point: /data
+        data_server_cache: reduced/ddcache
+        dummy_cache: /data/dummy_prods
+        data_server_url: http://oda-dda-interface:8000
+
+      jemx:
+        dispatcher_mnt_point: /data
+        data_server_cache: reduced/ddcache
+        dummy_cache: /data/dummy_prods
+        data_server_url: http://oda-dda-interface:8000
+
+  magic:
+    enabled: true
+    instruments:
+      magic:
+        data_server_url: http://oda-magic:8000
+        dummy_cache: /data/dummy_prods
+
+  polar:
+    enabled: true
+    instruments:
+      polar:
+        dispatcher_mnt_point:
+        data_server_cache:
+        dummy_cache: /data/dummy_prods
+        data_server_url: polar-worker
+        data_server_port: 8893
+
+  antares:
+    enabled: true
```

Comment:
I get that it is nice to be able to enable/disable plugins, but in this case it only enables/disables plugin configs, and what is installed in the dispatcher container still needs to be agreed with these values. Which is why I think both dispatcher container and  full plugin config are internal business of the entire oda application and can not be easily configurable, unless a more coherent way to enable/disable also plugin installation and backend deployment.